onnxruntime/core/providers/openvino/backend_manager.h

```diff
@@ -47,6 +52,8 @@ class BackendManager {
   std::map<std::string, std::shared_ptr<IBackend>> backend_map_;
   SubGraphContext subgraph_context_;
   GlobalContext global_context_;
+  EPCtxHandler ep_ctx_handle_{};
+  std::string openvino_sdk_version_{};
```

Comment:
can the return value be const here

Reply:
This is a statement to declare and initialize the EPCtxHandler object, and we are be manipulating the members of it so we cannot initialize it as const here.